src/client/src/commons/detail/meta/metaComponent.js

```diff
@@ -269,16 +315,19 @@ class MetaComponent extends React.Component {
             style={{
               flex: "1 1 100%",
             }}>
-            {this.getTextRow(
+            {this.getNumericTextRow(
               "coordinatesLV95",
               data.location_x + ", " + data.location_y,
             )}
-            {this.getTextRow(
+            {this.getNumericTextRow(
               "coordinatesLV03",
               data.location_x_lv03 + ", " + data.location_y_lv03,
```

Comment:
PP: X und Y werden hier mit einem Komma verbunden nur um in der _getNumericTextRow_ Funktion wieder gesplittet zu werden. Eventuell Rest parameters verwenden.

Reply:
REQ: Sehe ich auch so. [Rest Parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters) verwenden und sobalds mehrere Werte sind die `NumericFormat`'s mit `, ` joinen.